app/services/idv/steps/inherited_proofing/verify_wait_step_show.rb

```diff
@@ -58,9 +60,12 @@ def async_state_done(_current_async_state)
             inherited_proofing_save_user_pii_to_session!(form.user_pii)
             mark_step_complete(:verify_wait)
           else
+            idv_failure(form_response)
             mark_step_incomplete(:agreement)
```

Comment:
TBD: Probably mark :agreement step incomplete _before_ call to `idv_failure` which performs redirect_to?